package-lock.json

```diff
@@ -6,7 +6,7 @@
 	"packages": {
 		"": {
 			"name": "action-ros-ci",
-			"version": "0.2.7",
+			"version": "0.3.0",
```

Comment:
There's unfortunately a conflict here after I released `0.3.1`. Can you re-run `npm install` and amend/push to fix this? I think that should take care of it.

Reply:
Sure sorry this was just autogenerated so I checked it in.

Reply:
Yeah it's a bit annoying. Sometimes I forget to update it (or I guess it doesn't automatically get updated) when I bump the version.

Reply:
rebased to get `0.3.1`